src/Groups/pcgroup.jl

```diff
@@ -473,6 +472,110 @@ function pc_group(c::GAP_Collector)
   end
 end
 
+"""
+    letters(g::Union{PcGroupElem, SubPcGroupElem})
+
+Return the letters of `g` as a list of integers, each entry corresponding to
+a group generator.
+
+This method can produce letters represented by negative numbers. A negative number 
+indicates the inverse of the generator at the corresponding positive index.
+
+For example, as shown below, an output of -1 refers to the "inverse of the first generator".
+
+See also [`syllables`](@ref).
```

Comment:
```suggestion
See also [`syllables(::Union{PcGroupElem, SubPcGroupElem})`](@ref).
```
otherwise this may link to any function called `syllables`, e.g. to `syllables(::FPGroupElem)` which is not what we want here